src/main/java/com/google/sps/Database.java

```diff
@@ -0,0 +1,56 @@
+package com.google.sps;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.FirestoreOptions;
+import com.google.api.core.ApiFuture;
+import com.google.cloud.firestore.WriteResult;
+import com.google.cloud.firestore.DocumentReference;
+import com.google.cloud.firestore.SetOptions;
+
+import java.util.*;
+import java.io.*; 
+
+public class Database {
+    private Firestore db;
+
+    public void initialize() throws Exception {
+        GoogleCredentials credentials = GoogleCredentials.getApplicationDefault();
+        // Initialize instance of Firestore
+        FirestoreOptions firestoreOptions = FirestoreOptions.getDefaultInstance().toBuilder()
+                .setProjectId("spring21-sps-33").setCredentials(credentials).build();
+        db = firestoreOptions.getService();
+    }
+
+    public void addData(String filePath) throws Exception {
+        File fecData = new File(filePath);
+        
+        Scanner input = new Scanner(fecData); 
+
+        while (input.hasNextLine()){
+            String can = input.nextLine(); 
+            String[] stats = can.split("\\|"); 
+
+            CandidateStats canStats = new CandidateStats(stats[0], stats[18], stats[19], stats[4], stats[11], 
+            stats[12], stats[13], stats[17], stats[25], stats[26]);
+                
+            Map<String, Object> initialData = new HashMap<>();
+
+            initialData.put(filePath.substring(39, 48), canStats);
+
+            String candidateName = stats[1];
+
+            if (candidateName.contains("/")){
```

Comment:
Can you add a comment for why this change is necessary? Even though I think we all know it's for the VP situation, it's helpful to document edge cases like this for the future.